spec/firebase_app_distribution_api_client_spec.rb

```diff
@@ -52,11 +47,12 @@
       end
       upload_token = api_client.get_upload_token("app_id", fake_binary_path)
       binary_hash = Digest::SHA256.hexdigest(fake_binary_contents)
+      expect(api_client.instance_variable_get(:@auth_token))
```

Comment:
Should this have an expectation attached to it? e.g. `.to eq(something)` ?

Reply:
This was left over from a different way Alonso and I tried to make sure we were using auth token. Will remove.